STYLE_GUIDE.md

```diff
@@ -1,53 +1,45 @@
-# Composer Style Guide
+# Style and Conventions
 
-This guide serves as a rule-of-thumb for how code should be organized and formatted in Composer.
-For general python styling questions not covered in this document, consult Google's
-[Python Style Guide](https://google.github.io/styleguide/pyguide.html).
+### 2.1 Style Guide
 
-## 1. Code Formatting
+Composer generally follows Google's
+[Python Style Guide](https://google.github.io/styleguide/pyguide.html) for how to format and structure code.
+
+### 2.2. Code Formatting
 
 Composer uses the [yapf](https://github.com/google/yapf) formatter for general formatting,
-[isort](https://github.com/PyCQA/isort) to sort imports,
-[docformatter](https://github.com/myint/docformatter) to format docstrings, and
-[pylint](https://github.com/PyCQA/pylint) as the general style checker.
+[isort](https://github.com/PyCQA/isort) to sort imports, and
+[docformatter](https://github.com/myint/docformatter) to format docstrings.
 
 To (auto)format code, run
 
 ```bash
-yapf -rip .  # for yapf, auto-formats in place
-isort .  # for isort, auto-formats in place
-
-# for docformatter, auto-formats in place
-docformatter -ri --wrap-summaries 120 --wrap-descriptions 120 composer tests examples
-
-pylint composer tests examples  # for pylint, does NOT auto-format
+make style
+```
 
+To verify that styling and typing (see below) is correct, run:
+```bash
+make lint
 ```
 
 The configuration is stored in [pyproject.toml](pyproject.toml).
 
-All modified files must have their pylint errors addressed
-(eventually, the entire codebase will become pylint compliant).
-Please do NOT address pylint errors in unrelated, unmodified files,
-as that will clutter the PR.
 
-
-## 2. Type Annotations and Typechecking
+## 3. Type Annotations and Typechecking
 
 Composer aims to annotate all functions with type annotations (introduced in
 [PEP 526](https://www.python.org/dev/peps/pep-0526/). Type annotations help statically catch `TypeError` and
 `AttributeError` bugs, in addition to other benefits, as outlined in the PEP.
 
-Composer uses [pyright](https://github.com/microsoft/pyright)
+Composer uses [PyRight](https://github.com/microsoft/pyright)
```

Comment:
Pyright doesn't capitalize R. Also should probably just be lowercase to match usage below.